### PR TITLE
[android] enable 64-bit client builds

### PR DESCRIPTION
**Pull request by wkozyra95**

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -37,18 +37,7 @@ android {
...
-    ndk {
-      // We use different `abiFilters` for distributing
-      // to be able to bundle old, incompatible SDKs.
 
-      // WHEN_DISTRIBUTING_REMOVE_FROM_HERE
-      abiFilters 'armeabi-v7a', 'x86'
-      // WHEN_DISTRIBUTING_REMOVE_TO_HERE
-
-      /* UNCOMMENT WHEN DISTRIBUTING
-      abiFilters 'armeabi-v7a', 'x86', 'arm64-v8a', 'x86_64'
```
Comment (sjchmiela) [inline android/app/build.gradle]: Are you sure we can just remove this whole section? Shouldn't we keep that single line above to have only those 4 abis enabled?

Comment (wkozyra95) [reply android/app/build.gradle]: There are only binaries for those 4 platforms anywhere in our project so it should be fine

Comment (sjchmiela) [reply android/app/build.gradle]: So there is no possiblity of a developer specifying another set of libraries and trying to build an app with abi filter `'myabi-v10x'` which will build successfully, but then crash in runtime?

Comment (wkozyra95) [reply android/app/build.gradle]: it should fail during the build

**Review by sjchmiela — APPROVED** [review]
👍

**Review by ide — APPROVED** [review]

**Pull request merged**